snippets/terraform/certificates/main.tf

```diff
@@ -41,6 +41,12 @@ resource "azurerm_key_vault" "example" {
   tags = var.tags
 }
 
+resource "azurerm_role_assignment" "current_user" {
```

Comment:
make sure to add a note that this role offers "admin"-y permissions on the AKV. (similar to how there is a comment about firewall rule that allows ALL traffic from the internet).